component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/ComponentManager.java

```diff
@@ -885,7 +885,7 @@ protected boolean isContainerClass(final Filter filter, final String name) {
                         try {
                             return new File(decode(new URL(file.substring(0, separator)).getFile()))
                                     .getName()
-                                    .startsWith("beam-sdks-java-core-");
+                                    .contains("beam-sdks-java-core-");
```

Comment:
I assume it is due to the mutating packaging in your app which likely uses org.apache.beam.beam-sdks-java-core? If so can we use this as || startsWith instead of using contains which is more fragile?